docs/tools/terraform/howto/config-postgresql-provider.rst

```diff
@@ -0,0 +1,80 @@
+Use PostgreSQL® provider with Aiven's PostgreSQL service
```

Comment:
Title suggestion: Use PostgreSQL® provider for additional PostgreSQL configurations